tardis/plasma/properties/atomic.py

```diff
@@ -108,8 +109,17 @@ def calculate(self, atomic_data, selected_atoms):
 class IonizationData(ProcessingPlasmaProperty):
     name = 'ionization_data'
 
-    def calculate(self, atomic_data, selected_atoms):
+    def calculate(self, atomic_data, levels):
         if self.value is not None:
             return self.value
         else:
             return atomic_data.ionization_data
+
+class ZetaData(ProcessingPlasmaProperty):
+    name = 'zeta_data'
```

Comment:
@aoifeboyle yes, I think that sounds like a very good plan to make `ZetaData` standards compliant. 
they shouldn't have underscores, we need to check this out.